projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -976,7 +980,18 @@ public string BirthPhysicalLocationHelper
         public Dictionary<string, string> MotherPlaceOfBirth
         {
             get => GetPlaceOfBirth(Mother);
-            set => SetPlaceOfBirth(Mother, value);
+            set 
+            {
+                if (!String.IsNullOrEmpty(value["addressState"]) && !CodeExistsInValueSet(value["addressState"], VR.ValueSets.StatesTerritoriesProvinces.Codes))
```

Comment:
for consistent behavior across the library, this should attempt to set the coded value using the SetCodeValue which calls CreateCode and throws an argument exception if the code is not included in the value set